dune-project

```diff
@@ -1,132 +1,250 @@
 (lang dune 2.7)
```

Comment:
@mabiede 

The `dune-project` file seems to be formatted weirdly. When I run `make format` with the checked in `.ocamlformat` I get  the formatting before your change. (ocamlformat 0.20.1)

Reply:
test

Reply:
Thanks for the command, it seems that the VSCode extension is using dune to format dune and dune-project files.